cloudsync/sync/manager.py

```diff
@@ -347,14 +347,10 @@ def sync(self, sync: SyncEntry) -> bool:  # pylint: disable=too-many-branches,to
 
         for side in ordered:
             if not sync[side].needs_sync():
-                # if the changed side's path doesn't translate then ignore 'needs_sync' and process
-                # the change anyway (likely a rename to irrelevant directory)
-                # see test_cs_rename_folder_out_of_root
-                if self.translate(other_side(side), sync[side].path):
```

Comment:
i thought this was there to fix test_cs_rename_folder_out_of_root

Reply:
set_force_sync() is a more complete solution